cli/lib/kontena/machine/cloud_config/cloudinit.yml

```diff
@@ -19,27 +19,28 @@ write_files:
       fs.inotify.max_user_instances = 8192
 coreos:
   units:
-    - name: 00-eth.network
-      runtime: true
+    - name: 10-weave.network
```

Comment:
Removed the `10-weave.network` unit from the cloud-config.

I suppose it's not worth it to consider the use-case of re-generating the cloud-config for an existing node that was installed with an older cloud-config - you should just replace the entire node.